### PR TITLE
[test]: Add test_InterfaceAddRemoveSecondIpv4Address

### PR DESCRIPTION
**Pull request by stcheng**
This test is to cover cases where one router interface
has two IPs assigned to it.

Signed-off-by: Shu0T1an ChenG <shuche@microsoft.com>

### CODE REVIEW

File: tests/test_interface.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def test_InterfaceAddIpv4Address(self, dvs):
                     if fv[0] == "SAI_ROUTER_INTERFACE_ATTR_TYPE":
                         assert fv[1] == "SAI_ROUTER_INTERFACE_TYPE_PORT"
                     if fv[0] == "SAI_ROUTER_INTERFACE_ATTR_MTU":
-                        assert fv[1] == "1500"
+                        assert fv[1] == "9100"
```
Comment (lguohan) [inline tests/test_interface.py]: why do you need to modify this?

File: tests/test_interface.py [hunk 2]
```diff
@@ -88,6 +86,51 @@ def test_InterfaceChangeMtu(self, dvs):
...
 
+    def test_InterfaceAddRemoveSecondIpv4Address(self, dvs):
+        pdb = swsscommon.DBConnector(0, dvs.redis_sock, 0)
+        adb = swsscommon.DBConnector(1, dvs.redis_sock, 0)
+        cdb = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+
+        # assign second IP to interface
+        tbl = swsscommon.Table(cdb, "INTERFACE")
+        fvs = swsscommon.FieldValuePairs([("NULL", "NULL")])
+        tbl.set("Ethernet8|20.0.0.4/31", fvs)
```
Comment (lguohan) [inline tests/test_interface.py]: by default each test should be independent, so that when a test fails, we can debug it and re-run it.

**Comment by wendani** [comment]
retest this please

**Comment by wendani** [comment]
test this please

**Pull request closed without merging**